src/main/java/gift/domain/Category/CategoryService.java

```diff
@@ -1,56 +1,58 @@
-    package gift.domain.Category;
+package gift.domain.category;
 
-    import gift.global.exception.BusinessException;
-    import java.util.List;
-    import java.util.Optional;
-    import org.springframework.http.HttpStatus;
-    import org.springframework.stereotype.Service;
-    import org.springframework.transaction.annotation.Transactional;
+import gift.global.exception.BusinessException;
+import gift.global.exception.category.CategoryDuplicateException;
+import gift.global.exception.category.CategoryNotFoundException;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
 
-    @Service
-    public class CategoryService {
+@Service
+public class CategoryService {
 
-        private final JpaCategoryRepository categoryRepository;
+    private final gift.domain.category.JpaCategoryRepository categoryRepository;
 
-        public CategoryService(JpaCategoryRepository categoryRepository) {
-            this.categoryRepository = categoryRepository;
-        }
-
-        public List<Category> getCategories() {
-            List<Category> categories = categoryRepository.findAll();
-            return categories;
-        }
+    public CategoryService(gift.domain.category.JpaCategoryRepository categoryRepository) {
+        this.categoryRepository = categoryRepository;
+    }
 
-        public void createCategory(CategoryDTO categoryDTO) {
-            if (categoryRepository.existsByName(categoryDTO.getName())) {
-                throw new BusinessException(HttpStatus.BAD_REQUEST, "동일한 이름의 카테고리 존재");
-            }
+    public List<Category> getCategories() {
+        List<Category> categories = categoryRepository.findAll();
```

Comment:
저는 이런거 inline 시켜버리는 것을 선호합니당ㅎㅎ 취향 차이라서 꼭 적용안하셔도되어요!
단축키로는 Command + Option + N 사용하시면 됩니다!
```suggestion
        return categoryRepository.findAll();
```

Reply:
일단 말씀하신 스타일로 적용해보겠습니다 ㅎㅎ!